fyle_accounting_mappings/views.py

```diff
@@ -133,3 +133,20 @@ def get_queryset(self):
         return CategoryMapping.objects.filter(
             workspace_id=self.kwargs['workspace_id']
         ).all().order_by('source_category__value')
+
+
+class SearchDestinationAttributesView(ListCreateAPIView):
+    """
+    Search Destination Attributes View
+    """
+    serializer_class = DestinationAttributeSerializer
+
+    def get_queryset(self):
+        destination_attribute_value = self.request.query_params.get('destination_attribute_value')
+
+        assert_valid(destination_attribute_value is not None, 'query param destination_attribute_value not found')
+
+        destination_attributes = DestinationAttribute.objects.filter(
+            value__icontains=destination_attribute_value, workspace_id=self.kwargs['workspace_id']
```

Comment:
filter should be on attribute_type as well, plis to expect this from client

Reply:
+1

Reply:
👍